src/util/states.js

```diff
@@ -0,0 +1,54 @@
+const states = [
```

Comment:
I think the backend requires two letter state codes. Plenty of folks don't know the state codes so its good to have the full names for display, but we will need corresponding codes to send to the backend.